charts/openshift-metering/values.yaml

```diff
@@ -212,146 +212,206 @@ reporting-operator:
   spec:
     affinity: {}
     annotations: {}
-    authProxy:
-      authenticatedEmailsData: ""
-      authenticatedEmailsEnabled: false
-      authenticatedEmailsSecretName: reporting-operator-auth-proxy-authenticated-emails
-      cookieSecretName: reporting-operator-auth-proxy-cookie-seed
-      cookieSeed: ""
-      createAuthProxyClusterRole: true
-      createAuthenticatedEmailsSecret: true
-      createCookieSecret: true
-      createHtpasswdSecret: true
-      delegateURLsEnabled: false
-      delegateURLsPolicy: '{"/": {"group": "metering.openshift.io", "resource": "reports",
-        "namespace": "$(NAMESPACE)", "subresource": "export", "verb": "get"}}'
-      enabled: false
-      htpasswdData: ""
-      htpasswdSecretName: reporting-operator-auth-proxy-htpasswd
-      image:
-        pullPolicy: Always
-        repository: openshift/oauth-proxy
-        tag: v1.1.0
-      resources:
-        limits:
-          cpu: 500m
-          memory: 100Mi
-        requests:
-          cpu: 100m
-          memory: 50Mi
-      subjectAccessReviewEnabled: false
-      subjectAccessReviewPolicy: '{"group": "metering.openshift.io", "resource": "reports",
-        "namespace": "$(NAMESPACE)", "subresource": "export", "verb": "get"}'
+    labels: {}
+    nodeSelector: {}
+    tolerations: []
+    replicas: 1
+
+    image:
+      pullPolicy: Always
+      repository: quay.io/openshift/origin-metering-reporting-operator
+      tag: 4.2
+      pullSecrets: []
+
+    updateStrategy:
+      type: RollingUpdate
+
+    resources:
+      limits:
+        cpu: 1
+        memory: 500Mi
+      requests:
+        cpu: 500m
+        memory: 100Mi
+
     config:
       allNamespaces: false
-      awsAccessKeyID: ""
-      awsCredentialsSecretName: ""
-      awsSecretAccessKey: ""
-      createAwsCredentialsSecret: false
-      createClusterMonitoringViewRBAC: true
-      disablePrometheusMetricsImporter: "false"
-      enableFinalizers: "false"
-      hiveHost: hive-server:10000
-      leaderLeaseDuration: 60s
-      logDDLQueries: "false"
-      logDMLQueries: "false"
+      enableFinalizers: false
+      leaderLeaseDuration: "60s"
+
+      logDDLQueries: false
```

Comment:
I still prefer having a `logging` field that would store these, but if you think unnecessary, that's fine with me.

Reply:
I just feel like most components may not have many logging options, and they're all kind app specific so it seems generalizing this isn't as useful.